index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <title>ETCH-A-SKETCH</title>
+</head>
+<body>
+<div class = "menu">
+    <div id = "text">BLACK</div>
+    <div id = "dimension">DIMENSION : 16 X 16</div>
+</div>
+<div id = "container"></div>
+<div id = "defaultContainer"></div>
+<div id = "availableColors">
+    <button onclick = "edit()">EDIT DIMENSION</button>
+    <button onclick = "blue()">BLUE</button>
+    <button onclick = "red()">RED</button>
+    <button onclick = "yellow()">YELLOW</button>
+    <button onclick = "green()">GREEN</button>
+    <button onclick = "eraser()">ERAZER</button> 
```

Comment:
Consider binding the listeners from the javascript end instead of within the HTML.

Reply:
I'm used to this terrible was of creating functions
I'll try not to make such errors In the Future.